pipeline/outputs/s3.md

```diff
@@ -18,31 +18,31 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
+| json_date_format              | Specify the format of the date. Supported formats are _double_, _epoch_, _iso8601_ (eg: _2018-05-30T09:39:52.000681Z_) and _java_sql_timestamp_ (eg: _2018-05-30 09:39:52.000681_)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | iso8601                                 |
+| total_file_size               | Specifies the size of files in S3. Maximum size is 50G, minimim is 1M.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   | 100M                                    |
+| upload_chunk_size             | The size of each 'part' for multipart uploads without gzip. Max: 50M; for multipart uploads with gzip. Max: 50M                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | 5,242,880 bytes                         |
```

Comment:
language nit/suggestion: "The size of each 'part' for multipart uploads.  For uncompressed uploads, the max is 50M. For compressed uploads, the max is 5G. This is the size of the buffered data before compression is applied, so the actual uploaded chunk will be smaller. The min upload_chunk_size in all cases is restricted in the S3 API to 5,242,880 bytes. If after compression, the chunk is smaller than that size, the plugin will upload the chunk using the PutObject API instead."

Something like this, we need more explanation for the user. Another idea is that we should instead not make this part of the doc too long and complicated and have a special section further down to discuss multipart compression. And then in this part just note that the user should read that section. Let's discuss next week.

Reply:
Will do